starter-bots/kotlin/ReadMe.md

```diff
@@ -0,0 +1,23 @@
+# Kotlin Sample Bot
+
+Kotlin is a statically typed programming language that runs on the Java virtual machine. It is 100% interoperable with Java, but was built to be more expressive, concise and safer.
+
+## Learn Kotlin
+
+The official documentation and tutorials can be found https://kotlinlang.org/
+
+
+## Environment Setup
+
+Install the [Java Development Kit](http://www.oracle.com/technetwork/java/javase/downloads/index.html) and [Maven](https://maven.apache.org/)
+
```

Comment:
This part is from the Java bot readme. If it is directly compatible with the Kotlin version, i believe it is a good idea to nudge them to get intelliJ
```
Install the Java SE Development Kit 8 for your environment here: http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html
Make sure JAVA_HOME system variable is set, Windows 10 tutorial here: https://www.mkyong.com/java/how-to-set-java_home-on-windows-10/
Install IntelliJ IDEA here: https://www.jetbrains.com/idea/download/
The community edition is free.
```